src/core/utils/platform/f4xx/timef4xx.cpp

```diff
@@ -0,0 +1,22 @@
+/**
+ * STMF3xx implementation of the time functionality. This code is designed
```

Comment:
This has nothing to do with what you were doing, but I'd appreciate fixing this documentation